Client/Frontend/Fakespot/Views/FakespotFadeLabel.swift

```diff
@@ -0,0 +1,43 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+/// A label that adds a fade at bottom to indicate more content is available to user
+public class FakespotFadeLabel: UILabel {
+    private let fadePercentage: Double = 0.5
+    private let gradientLayer = CAGradientLayer()
+    private let transparentColor = UIColor.clear.cgColor
+    private let opaqueColor = UIColor.black.cgColor
+    private let maskLayer = CALayer()
+
+    private var bottomOpacity: CGColor {
+        if isShowingFade {
+            return UIColor(white: 0, alpha: 0).cgColor
```

Comment:
Minor, there's [UIColor.clear](https://developer.apple.com/documentation/uikit/uicolor/1621945-clear) in the standard library.